content/guides/advanced/verified-builds.md

```diff
@@ -23,6 +23,36 @@ verified builds for their programs on Solana. We will cover what verified builds
 are, how to use them, special considerations, and best practices to ensure the
 authenticity of your program onchain.
 
+# How to Verify a Program
+
+## Table of Contents
+
+- [What are verified builds?](#what-are-verified-builds)
+- [Why should I use verified builds?](#why-should-i-use-verified-builds)
+  - [Benefits](#why-should-i-use-verified-builds)
+- [How do I create verified builds?](#how-do-i-create-verified-builds)
+  - [Install Docker and Cargo](#install-docker-and-cargo)
+  - [Install the Solana Verify CLI](#install-the-solana-verify-cli)
+  - [Prepare project](#prepare-project)
+  - [Building Verifiable Programs](#building-verifiable-programs)
+  - [Remote Verification](#remote-verification)
+- [How to verify your program when its controlled by a Multisig](#how-to-verify-your-program-when-its-controlled-by-a-multisig-like-squads)
+  - [Build and Deploy](#1-build-the-verifiable-program)
+  - [Verify Repository](#3-commit-and-verify-against-repository)
+  - [Transfer to Multisig](#4-transfer-program-authority-to-multisig)
+  - [Export and Submit](#5-export-pda-transaction)
+  - [Program Updates](#8-updating-the-program-optional)
+- [Verify from docker image](#verify-from-docker-image)
+- [Example verified build](#example-verified-build)
+- [Some programs that are already verified](#some-programs-that-are-already-verified)
+  - [Phoenix](#phoenix)
+  - [Squads V3](#squads-v3)
+  - [Drift V2](#drift-v2)
+  - [Marginfi V2](#marginfi-v2)
+- [Security + Disclaimer](#security--disclaimer)
+
+[Insert rest of document here...]
```

Comment:
a TOC is already generated by the website, so adding it in here just duplicates the same thing

```suggestion
```